spec/rspec/rails/fixture_file_upload_support_spec.rb

```diff
@@ -25,6 +25,12 @@ module RSpec::Rails
       end
     end
 
+    context 'ActionDispatch::TestProcess on Rails 6.1' do
+      it 'can read the file_fixture_path attribute' do
+        expect(FixtureFileUploadSupport::RailsFixtureFileWrapper).to respond_to(:file_fixture_path)
```

Comment:
This explains the how, but not the why, any chance of an integration spec for 6.1? Or do some of our existing integration specs fail on 6.1 for this reason?

Reply:
> Or do some of our existing integration specs fail on 6.1 for this reason?

Exactly, the wrapped class `ActionDispatch::TestProcess` requires access to this attribute as of Rails 6.1, otherwise fixture_file_upload bombs out. As to why that class needs to be wrapped in the first place, I have no idea. 

I'm not sure what the convention is for upcoming releases: is 6.1 running on a branch somewhere? Is it trivial for me to set that up locally?